recipes/x86_64_level/meta.yaml

```diff
@@ -0,0 +1,71 @@
+{% set number = 0 %}
+
+package:
+  name: microarch-level-split
+  version: {{ level }}
+
+build:
+  number: {{ number }}
+  string: {{ number }}_{{ microarchitecture }}
+  noarch: generic
+  run_exports:
+    strong:
+      - _{{ family }}-microarch-level >={{ level }}
```

Comment:
This should be moved to the `{{ family }}-microarch-level` output, right?

Reply:
This way, it's added for both outputs.

Reply:
No, it would only be added to `microarch-level-split` if that output existed.
Same with the `noarch: generic`. We should put those in the outputs too.
(See the `linux-64` subdir in https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=819077&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=933 )